deployment/docker/Makefile

```diff
@@ -170,7 +172,7 @@ package-via-docker-386:
 	GOARCH=386 $(MAKE) package-via-docker-goarch-nocgo
 
 remove-docker-images:
-	docker image ls --format '{{.Repository}}\t{{.ID}}' | awk '{print $$2}' | xargs docker image rm -f
```

Comment:
@atykhyy This line seems to be a breaking change to me, could you update the PR?

Reply:
Why is it a breaking change? Before the change, `ls --format` emitted two tokens per line (image name and id) and then `awk` threw away the first token in each line passed to xargs. After the change, `ls --format` just emits the second token (image id) directly.

Reply:
Ah, yes, you are right.

But considering that change is about adding ability to override `docker image ls` with buildah is it expected to leave `docker image` here?

Reply:
This change is only about reducing a bit of repetition. I don't think it's worth introducing variables if they're only used once. For context, overriding `docker` with `buildah` is unfortunately more complicated than substituting `buildah` for `docker` everywhere. Sometimes subcommand names or command line arguments differ: e.g. in this line, one would have to say `buildah images -q | xargs buildah rmi`. Sometimes there is no one-liner equivalent at all, e.g. `buildah run` works on an pre-existing container unlike `docker run` which creates and optionally destroys one.